wpilibj/src/main/java/edu/wpi/first/wpilibj/motorcontrol/NidecBrushless.java

```diff
@@ -45,7 +44,7 @@ public NidecBrushless(final int pwmChannel, final int dioChannel) {
     m_pwm = new PWM(pwmChannel);
     SendableRegistry.addChild(this, m_pwm);
 
-    HAL.report(tResourceType.kResourceType_NidecBrushless, pwmChannel + 1);
+    HAL.reportUsage("NidecBrushless", pwmChannel, "");
```

Comment:
Why doesn't this report dioChannel like C++ does?

Reply:
Inconsistency in the original implementations.  Most likely Nidec will be removed anyway.